CrossSectionAnalysis/CrossSectionAnalysis.py

```diff
@@ -659,6 +678,112 @@ def moveSliceViewToMaximumSurfaceAreaStenosis(self):
     if point == -1:
         return
     self.ui.moveToPointSliderWidget.setValue(point)
+  
+  def resetLumenRegions(self):
+    if self._lumenRegions:
+      self._lumenRegions.clear()
+    self.ui.surfaceInformationLabel.clear()
+    # Don't; is delegated to parameter node.
```

Comment:
If you mean the specific line

# self.ui.surfaceInformationGoToToolButton.setChecked(False)

it means 'let the parameter node handle the checked status of the GoTo tool button; it's unchecked by default, don't uncheck it because we are resetting all widgets to defaults, so that we don't have to click it again'. The commented lines are left there for me in the future.

Reply:
I just did not understand the comment.
In general, commented out lines should not be left in the code. If you want to get back to it later then commit the code and then remove it in a follow-up commit. You can still find the removed code in the git history.